low-level/files/opt/seagate/sspl/setup/generate_sspl_bundle/sspl_bundle_generate

```diff
@@ -135,16 +147,16 @@ class SSPLBundleGenerate(object):
             tmpFile.close()
             raise SSPLBundleError(1, msg)
 
-        print("SSPL Support Bundle Files Uploaded Successfully..!")
-        print("SSPL Support Bundle Location: %s" % self.localTempPath+file_name)
+        logger.info("SSPL Support Bundle Files Uploaded Successfully..!")
```

Comment:
Can we print those all messages on the terminal along with adding them into the log file @cdeshmukh ?
This message from the terminal will show that the support bundle generation is done.
@gargpalak048

Reply:
Ack.
Pass an 'console' arg to the support bundle script. then the logs will be logged both in log file & console.